vars/Debian.yml

```diff
@@ -1,3 +1,4 @@
 ---
 __unattended_origins_patterns:
-  - 'origin=Debian,archive=${distro_codename},label=Debian-Security'
+  - 'origin=Debian,archive=stable,label=Debian-Security'
+  - 'origin=Debian,archive=oldstable,label=Debian-Security'
```

Comment:
By the time Debian release a new version (every 5 years), we will have upgraded the machine many time.